### ISSUE TITLE
Support redirect url in Client

### ISSUE THREAD

**Issue by kpritam** [bilal-fazlani/embedded-keycloak#8]
case class Client(name: String,
                    clientType: String = "public",
                    authorizationEnabled: Boolean = false,
                    passwordGrantEnabled: Boolean = true,
                    resourceRoles: Set[String] = Set.empty)

Should it also contain redirect url's?

**Comment by bilal-fazlani** [comment]
Yes, good point. Since this is a testing library, IMO,  setting it to * in the code (in [this](https://github.com/bilal-fazlani/embedded-keycloak/blob/master/src/main/resources/create-client-request.json) file) would be simple rather than adding a new field...
